resolvers/webpack/test/config.js

```diff
@@ -91,4 +91,13 @@ describe("config", function () {
         .and.equal(path.join(__dirname, 'files', 'some', 'goofy', 'path', 'foo.js'))
   })
 
+  it("finds the real config when its not an absolute path, ignoring node_modules packages on the way", function () {
+    var reexportingFile = path.join(__dirname, 'files', 'node_modules', 'module-with-reexports', 'lib', 'index.js')
+    var settings = {
+      config: 'webpack.config.js',
+    }
+
+    expect(resolve('./reexported-module', reexportingFile, settings)).to.have.property('path')
+      .and.equal(path.join(__dirname, 'files', 'node_modules', 'module-with-reexports', 'lib', 'reexported-module.js'))
```

Comment:
Can you clarify exactly how this fails when your fix is not included?

Reply:
The webpack resolver tried to load `webpack.config.js` relative to the closest directory with `package.json` - but that might be some node_module and there is obviously no `webpack.config.js` published and even if there was using it wouldnt give proper results. Webpack uses a single config for its resolution algorithm.